<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarMaintain</class>
 <widget class="QDialog" name="CarMaintain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加保养信息</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>序号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>60</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>车牌号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保养类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>68</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>现公里数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCarID">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStation">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditType">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrice">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDistance">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>210</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>责任人</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInCharge">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditID</tabstop>
  <tabstop>lineEditCarID</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>lineEditStation</tabstop>
  <tabstop>lineEditPrice</tabstop>
  <tabstop>lineEditType</tabstop>
  <tabstop>lineEditDistance</tabstop>
  <tabstop>lineEditInCharge</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
